packages/form-js-viewer/src/features/repeatRender/index.js

```diff
@@ -1,8 +1,10 @@
 import { RepeatRenderManager } from './RepeatRenderManager';
+import { FileRegistry } from '../../render/FileRegistry';
 
 export const RepeatRenderModule = {
   __init__: ['repeatRenderManager'],
   repeatRenderManager: ['type', RepeatRenderManager],
+  fileRegistry: ['type', FileRegistry],
```

Comment:
Just for future reference, we're registering this twice which I guess was due to a timing issue, but the modules access eachother. So probably here you needed either a `__depends__` or to re-order some things.